backend/graphql/resolvers/schoolResolvers.ts

```diff
@@ -12,6 +12,12 @@ const schoolService: ISchoolService = new SchoolService(userService);
 
 const schoolResolvers = {
   Query: {
+    school: async (
+      _req: undefined,
```

Comment:
first arg of a resolver is always the "parent" obj which allows you to return nested objects (via resolver chain) - i believe it can be named anything, but usually smth like `parent`, `obj`, or `_` by convention

https://www.apollographql.com/docs/apollo-server/data/resolvers/#resolver-arguments

Reply:
oh nice. thank you!